docs/BreakingChanges/049 ResolveAssemblyReference task now warns on dependencies with the wrong architecture.md

```diff
@@ -0,0 +1,26 @@
+## 49: ResolveAssemblyReference task now warns on dependencies with the wrong architecture
```

Comment:
{
  "Id": "49",
  "Title": "ResolveAssemblyReference task now warns on dependencies with the wrong architecture",
  "Details": "The task emits a warning, MSB3270, which indicates that a reference or any of its dependencies does not match the app's architecture. For example, this occurs if an app that was compiled with the anycpu option includes an x86 reference. Such a scenario could result in an app failure at run time (in this case, if the app is deployed as an x64 process).",
  "Suggestion": "There are two areas of impact:\n\n* Recompilation generates warnings that did not appear when the app was compiled under a previous version of MSBuild. However, because the warning identifies a possible source of runtime failure, it should be investigated and addressed.\n* If warnings are treated as errors, the app will fail to compile.",
  "Link": "https://msdn.microsoft.com/en-us/library/dn458356(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis